Lib/test/test__opcode.py

```diff
@@ -64,5 +64,27 @@ def test_stack_effect_jump(self):
                     self.assertEqual(nojump, common)
 
 
+class SpecializationStatsTests(unittest.TestCase):
+    def test_specialization_stats(self):
+        STAT_NAMES = ['specialization_success', 'specialization_failure',
+                      'hit', 'deferred', 'miss', 'deopt', 'unquickened']
+
+        stats = _opcode.get_specialization_stats()
+        if stats is not None:
+            self.assertIsInstance(stats, dict)
+            self.assertCountEqual(
+                stats.keys(),
+                ['load_attr', 'load_global', 'binary_subscr'])
```

Comment:
This is going to be a nuisance, as we add more specializations.
`[op[:-len("_ADAPTIVE")].lower() for op in opcode._specialized_instructions if op.endswith("_ADAPTIVE")]`